### PR TITLE
Adds Garbage receptacle by the kitchen and fixes a few map bugs.

### LINKED ISSUE #7840: No external access for new arrivals airlock
**Problem Description**:
The new cycling airlock at arrivals doesn't have a button to activate it from the exterior.

**What did you expect to happen**: 
To be able to enter from the exterior.

**What happened instead**:
Unable to enter.

**Why is this bad/What are the consequences:**
Forced to break/create entrance or go to a different airlock.

**Steps to reproduce the problem**:
Attempt to enter arrivals airlock from exterior.

### LINKED ISSUE #7895: Obsolete science console in bridge
#**Problem Description**:
What is the problem?
There's a science console left over in the bridge, while the science outpost has been removed.
**What did you expect to happen**: 
Why do you think this is a bug?
- It's a simple oversight
**What happened instead**:
How is what happened different from what you expected?
- It's a simple oversight
**Why is this bad/What are the consequences:**
Why do you think this is an important issue?
- No consequences, just a simple oversight
**Steps to reproduce the problem**:
The most important section. Review everything you did leading up to causing the issue.
- Remove the console
**Possibly related stuff (which gamemode was it? What were you doing at the time? Was
anything else out of the ordinary happening?)**: Anything else you can tell us.

### PR DESCRIPTION
**Pull request by SamHPurp**
As requested by a bunch of nerds. https://nanotrasen.se/forum/topic/10787-add-a-disposal-unit-next-to-the-kitchen/

image

This is a re-skinned crate as a garbage receptacle... like the one on Mining Outpost. This is NOT a disposal unit due to the potential for grief.

Also teeny fix to access levels on the Arrivals airlock... that happens when I have to fix conflicts 7 times.... RIP.

Fixes #7895
Fixes #7840

:cl: Purpose2
add: BoxStation: Adds a garbage receptacle by the kitchen. Don't forget to bus your plates.
fix: BoxStation: Fixes an inappropriate access restriction on one part of an airlock, and ensures the buttons are visible.
tweak: BoxStation: Changes the now erroneous Science Shuttle Console on the bridge to a Labor Camp Shuttle Console.
/ :cl:

### CODE REVIEW

**Comment by taukausanake** [comment]
Finally people can stop leaving their trash everywhere on the kitchen table.

Sidenote: is it just me or is that table one tile shorter than it used to?

**Comment by SamHPurp** [comment]
@taukausanake just you. I didn't touch it.

There was a potted plant in that corner before.

**Comment by taukausanake** [comment]
@SamHPurp Keep the plant and have the bin to the left of it? No reason to really but I think it would be nice

**Comment by SamHPurp** [comment]
@taukausanake That'd be fine. I was just working off the original suggestion. ;)

image

**Comment by taukausanake** [comment]
@SamHPurp You know that's not the only place they leave trash, even if the plant is not there

**Comment by Fox-McCloud** [comment]
I fixed the title and description on your PR.

😉 

:us:

**Comment by Citinited** [comment]
Take your 'Muricanism someplace else @Fox-McCloud 👎 ❌ 💯 

👍 to this PR, but ONLY in its "bin" state

**Comment by Birdtalon** [comment]
It's a bin 😠 !!

**Comment by SamHPurp** [comment]
> I fixed the title and description on your PR.

_Lodges complaint with the embassy._

**Comment by Ralta** [comment]
Do not merge until it is named correctly.

**Comment by FalseIncarnate** [comment]
:+1: This has been something that bothers me whenever I play chef. People either just drop their trash on the floor and leave or throw it back on the counter, potentially getting in the way of other foods.

This also reminds me I need to make it so you can wash plates to remove those crumbs and make them usable again... though hardly anyone actually uses plates and most foods magically create a trash plate object... Gonna probably have to change that.

**Comment by taukausanake** [comment]
@FalseIncarnate I was LITERALLY going to comment on the same thing about magical bluespace plates coming out of no where the last time I posted. Just didn't think it was worth the trouble pointing out an obvious problem

Brilliant idea for a refactor: make containers that are how you cook food on the machines. Things like plates or pans, pots. Stuffing the food inside the container means you could cook multiple things at the same time and not have them mix

**Comment by FalseIncarnate** [comment]
Worth considering, but definitely beyond scope for this PR.

One minor issue I forsee is someone stealing the bin (looking at you cargo) or it getting moved around since it isn't anchored like a disposal bin would be. Not a huge issue though, you can always have engineering or atmos install a real disposals if this one goes missing.

**Comment by taukausanake** [comment]
Maybe make them construct-able with metal? I think five sheets should be fine, maybe three? Also I don't know if you can fill it with a garbage bag to better empty it out (I know bags can auto fill from a tile but the concept still) but all this in another PR

**Comment by SamHPurp** [comment]
@FalseIncarnate I anchored it actually. I was really on the fence about doing so, but did it.

**Comment by FalseIncarnate** [comment]
Is the one on the mining outpost also anchored? If so, I guess this is fine, but if not, let's un-anchor this one and make them roll with the threat of garbage theft.

**Comment by Fox-McCloud** [comment]
>It's a bin 

All maintainers and coders are all American, not to mention we've generally stuck with American spelling for most things--for all intents and purposes the codebase is American 😉

**Comment by SamHPurp** [comment]
@Fox-McCloud 

I mean, the actual coded item in this `american codebase` says bin ;)

**Comment by SamHPurp** [comment]
@taukausanake the arguement is over can vs bin not garbage vs trash

**Comment by swordsith** [comment]
insta-merge pls

**Comment by taukausanake** [comment]
@SamHPurp 
> it's a trash bin
It's smaller than a trash or garbage can
trash or garbage **bin** is proper

**Comment by SamHPurp** [comment]
@taukausanake we agree. I'm pushing for bin. Fox is insisting on Can.

I'm not sure why you think we disagree.

**Comment by taukausanake** [comment]
It sounded like you thought I thought the argument was over trash or garbage. I prefer bin because it's not a can like traditional AMERICAN trash/garabge cans

**Comment by KasparoVy** [comment]
:rofl: Count how many times you see "colour" in the code, then "color" on the next line

**Comment by FalseIncarnate** [comment]
@KasparoVy that is actually due to BYOND having a built-in var "color", so some old things were updated to use that var and others have to use "colour" to avoid conflicts

**Comment by KingPhilipIII** [comment]
I'm firmly in support of the can argument. Take your bins elsewhere.

File: _maps/map_files/cyberiad/cyberiad.dmm [hunk 1]
```diff
@@ -3503,6 +3503,7 @@
 "bps" = (/obj/structure/stool/bed/chair{dir = 1},/obj/machinery/door_control{id = "bridge blast"; name = "Bridge Blast Door Control"; pixel_x = 28; pixel_y = -2; req_access_txt = "19"},/obj/machinery/keycard_auth{pixel_x = 29; pixel_y = 8},/turf/simulated/floor/plasteel,/area/bridge)
 "bpt" = (/obj/structure/stool/bed/chair{dir = 1},/turf/simulated/floor/plasteel,/area/bridge)
 "bpu" = (/obj/structure/table/reinforced,/obj/item/weapon/book/manual/sop_command,/obj/item/device/aicard,/obj/item/device/multitool,/turf/simulated/floor/plasteel{icon_state = "blue"; dir = 8},/area/bridge)
+"bpv" = (/obj/structure/closet/crate/bin{anchored = 1; name = "Garbage bin"; pixel_y = 8},/turf/simulated/floor/plasteel{dir = 1; icon_state = "whitecorner"},/area/hallway/primary/starboard/west)
```
Comment (Fox-McCloud) [inline _maps/map_files/cyberiad/cyberiad.dmm]: you know what you must do.

Comment (SamHPurp) [inline _maps/map_files/cyberiad/cyberiad.dmm]: reeee, This is on z5 too... and the `path` is `/bin`.... god damn.

File: code/game/objects/structures/crates_lockers/crates.dm [hunk 2]
```diff
@@ -434,12 +434,41 @@
 		return newgas
 
 
-/obj/structure/closet/crate/bin
-	desc = "A large bin."
-	name = "large bin"
+/obj/structure/closet/crate/can
+	desc = "A large can, looks like a bin to me."
```
Comment (taukausanake) [inline code/game/objects/structures/crates_lockers/crates.dm]: lol so salty, but I still agree with bin

Comment (Citinited) [inline code/game/objects/structures/crates_lockers/crates.dm]: We need more things like this in the codebase

**Comment by FalseIncarnate** [comment]
Map conflicts. Just z5 though.

**Comment by SamHPurp** [comment]
Yankification complete, as well as conflicts resolved.

**Comment by SamHPurp** [comment]
Also changed the science shuttle console to a labor shuttle console on the bridge... this'll work until we also remove labor camp ;)

**Comment by AffectedArc07** [comment]
*>Legitamate PR devolves into armecian/english debate*

**Comment by SamHPurp** [comment]
More conflicts

image

**Comment by SamHPurp** [comment]
@Fox-McCloud fully updated... EVEN THOUGH EVERYONE HAS A RECYCLING BIN NOT A CAN....

image

And last I checked Bill Gates was American.

Rant aside, please merge.

**Comment by SamHPurp** [comment]
whoops

**Pull request merged**